.github/workflows/generate-release-notes.yml

```diff
@@ -39,27 +34,14 @@ jobs:
           versionFile="./eng/Versions.props"
           release_version=$(perl -ne '/<VersionPrefix>([^<]*)/ && print $1' $versionFile)
           release_version_label=$(perl -ne '/<PreReleaseVersionLabel>([^<]*)/ && print $1' $versionFile)
-          major_minor_version=${release_version%.*}
-
-          if [[ "$TEST_RUN" == "true" ]]; then
-            qualified_release_version="test-$release_version"
-          else
-            version_url="https://aka.ms/dotnet/diagnostics/monitor${major_minor_version}/release/dotnet-monitor.nupkg.version"
```

Comment:
Can we use `aka.ms/dotnet/diagnostics/monitor${major_minor_version}/release/dotnet-monitor-win-x64.zip.version` rather than changing how all of this works?